docs/t-sql/functions/char-transact-sql.md

```diff
@@ -176,8 +177,10 @@ single_byte_representing_complete_character single_byte_representing_complete_ch
 ｼ                                           ｼ                                         
 ```
 
-### F. Using CHAR to return multibyte characters  
-This example uses the an integer and hex values in the valid range for ASCII. However, the CHAR function returns NULL because the parameter represents only the first byte of a multibyte character.
+### F. Using CHAR to return multibyte characters
+This example uses integer and hex values in the valid range for Extended ASCII. However, the CHAR function returns NULL because the parameter represents only the first byte of a multibyte character.
+A `CHAR(2)` double-byte character cannot be composed from constituent first and second `CHAR(1)` bytes in the way that

```

Comment:
This is too convoluted and needs to be explained in a way that's less cryptic for the average user. Please optimize for that.

Reply:
I've updated that with a slightly broader, less wordy assertion.